service/gradle.properties

```diff
@@ -1,6 +1,13 @@
 # Required due to bug in google-java-format plugin. See https://github.com/diffplug/spotless/issues/834
-org.gradle.jvmargs=--add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED \
+org.gradle.jvmargs= \
+  --add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED \
   --add-exports jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED \
   --add-exports jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED \
   --add-exports jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED \
   --add-exports jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED
+
+# Liquibase properties
+dbUri=jdbc:postgresql://127.0.0.1:5432/catalog_db
```

Comment:
Is there a reason to store the properties here instead of putting them the `build.gradle` file? I think it would be better to only have them in the `build.gradle` unless there's a benefit to using a properties file too

Reply:
No particular reason! I thought it might be clearer to have that separation but it also seems to break things